libs/common/src/components/osmosis-screen-container/index.tsx

```diff
@@ -9,22 +10,29 @@ import { OsmosisHeader, OsmosisHeaderProps } from "../osmosis-header";
 export interface OsmosisScreenContainerProps
   extends Omit<OsmosisHeaderProps, "onClose"> {
   children: ReactNode;
+  backgroundStyle?: BackgroundStyle;
 }
 
+const getBackGroundStyles = (style: BackgroundStyle) => {
+  return {
+    ...(style.color && { backgroundColor: style.color }),
+    ...(style.color && { backgroundSize: style.size }),
+    ...(style.color && { backgroundPosition: style.position }),
+    ...(style.color && { backgroundImage: `url(${style.image})` }),
+    ...(style.blendMode && { backgroundBlendMode: style.blendMode }),
```

Comment:
That's not correct (see the copy-pasted style.color). And also completely unnecessary:
```
backgroundColor: style.color,
backgroundSize: style.size,
```
etc. should already work as expected.

Reply:
good eye, what do you mean unnecessary? I just wanted to add a param if present.

Reply:
Like I said, `{ backgroundColor: style.color }` should already work correctly if `style.color` is `undefined` since `foo[whatever]` is also `undefined` if `foo` doesn't contain the key `whatever`. Same behavior.

Reply:
Right but if you see what I did, the idea is to override the styles with the ones in the screen, for example theme. background can have defined some props but welcome can have others (or not), what would happen if theme.background has a color defined but welcome.background doesn't? it would set backgroundColor:undefined

Reply:
Yes, but IMHO, that doesn't make sense. Especially, with your current implementation, there wouldn't even be a way to unset `theme.background.color`. This again makes an implicit coupling (one has to know that `theme.background` leaks into `welcome.background`, even if you explicitly set some stuff to undefined) and some cases - e.g. the one I mentioned - aren't event possible. So IMHO, it makes more sense to have `theme.background` and `theme.welcome.background` completely independent. Yes, this means that you'd have to explicitly copy the color if you want to use it in both places, but I'd rather have that instead of implicit overriding.

Reply:
ok so I will do it so that there is only one `style` it will take style from `theme.background` by deault but if there is a backgroundStyle prop then take that instead, so that they don't mix